src/makeRobotAccountant.js

```diff
@@ -5,8 +5,21 @@
  * @return {function}
  */
 
-function makeRobotAccountant() {
-  // write code here
+function makeRobotAccountant(a) {
+  let counter = 0;
+
+  return (firstNum) => {
+    return (secondNum) => {
+      counter++;
+
+      const isCallCounterBiggerThanThree = counter > 3;
+      const isCallCounterOdd = counter % 2 === 0;
```

Comment:
nice approach, just a naming suggestion
```suggestion
      const isLimitExceeded = counter > 3;
      const isCallCounterOdd = counter % 2 === 0;
```